refl1d/webview/client/package.json

```diff
@@ -9,12 +9,13 @@
   },
   "scripts": {
     "dev": "vite",
-    "build": "vite build --emptyOutDir -m development",
+    "build": "vite build --sourcemap --emptyOutDir -m development",
```

Comment:
I don't think we want to build sourcemaps all the time - it will slow things down and bloat the distributed packages. You can always do `npm run build -- --sourcemap` locally to test things out, and there's a flag to the python helper for doing that too.